### PR TITLE
Adds instructor to Wordpress data loader

### PR DESCRIPTION
**Pull request by rehan99000**
**Description:**
This PR updates wordpress data loader to store instructor information from wordpress.

**JIRA:**
https://edlyio.atlassian.net/browse/EDLY-2598

**Related wordpress PR:**
https://github.com/edly-io/edly-wp-plugin/pull/345

### CODE REVIEW

File: course_discovery/apps/course_metadata/data_loaders/api.py [hunk 1]
```diff
@@ -953,6 +953,21 @@ def _load_data(self, page):
         response = self._make_request(page)
         self._process_response(response)
 
+    def _add_course_instructors(self, course_instructors, course_run):
+        """
+        Create and add instructors to a course run.
+        """
+        for course_instructor in course_instructors:
+            course_instructor['partner'] = course_run.course.partner
+            instructor, ___ = Person.objects.get_or_create(
```
Comment (zubair-arbi) [inline course_discovery/apps/course_metadata/data_loaders/api.py]: @rehan99000 We haven't considered a case when the information of the instructor is changed. In current scenario we will be adding old instructor (with modified info) as a new instructor which is not a desired approach.

CC: @asadmanzoor93

**Review by zubair-arbi — CHANGES_REQUESTED** [review]

File: course_discovery/apps/course_metadata/data_loaders/api.py (hunk 1 above)
Comment (rehan99000) [reply course_discovery/apps/course_metadata/data_loaders/api.py]: I think we can update `get` params to instructor id and add/update according to that. will update it as soon as panel work is done

Comment (rehan99000) [reply course_discovery/apps/course_metadata/data_loaders/api.py]: method updated. now it updates the instructor if instructor is updated as well

**Review by zubair-arbi — APPROVED** [review]
LGTM 👍

**Pull request merged**